PacketDetection/ScannerRegistry.cs

```diff
@@ -382,12 +405,6 @@ private void DeclareScanners()
                 PacketSource.Server,
                 (packet, _) => packet.PacketSize == 156 && BitConverter.ToUInt16(packet.Data, Offsets.IpcData + 8) == 16533);
             //=================
-            RegisterScanner("AddStatusEffect", "Please use Dia.",
```

Comment:
Why does this need to be removed? If it's because it's job-specific you can just click the skip button on the UI.

Reply:
If it's the same scanner then it should be corrected, not changed to what it's currently scanning for. At any rate, if this scanner is returning the value for EffectResult then that's a bug; I'll fix that later. Outside the scope of this PR.